src/components/ui/EmptyState.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import faker from 'faker'
+import { storiesOf } from '@storybook/react'
+
+import EmptyState from './EmptyState'
+
+storiesOf('UI Components|EmptyState', module)
+  .add('basic usage', () => (
+    <EmptyState heading='No jobs at the moment...'>
+      <span>
+        Please check the Jobs Board later. In the meantime make sure your <a
+          href='https://www.linkedin.com/in/'
+          target='_blank'
+          rel='noopener noreferrer'
+        >profile is up-to-date.</a>
```

Comment:
Moved the `.` outside the link as per Figma mocks.
```suggestion
        >profile is up-to-date</a>.
```